openmrs/apps/reports/RADET/sql/index_testing.sql

```diff
@@ -1,33 +1,35 @@
-	0 as "Index Classification / Classification de l'indice",
-	0 as "HTS Entry/ entrée  (Testing Modality/ modalités de test)",
-	0 as "Date of contact notification / Date de notification de contact",
-	0 as "Method of contact notification/ Méthode de notification de contact",
-	0 as "Outcomes of notifictaion / Résultats de la notification",
-	0 as "HTS Client ID (EID or HEI Number)",
-	0 as "Actual date of visit for HTS/ Date réelle de visite pour HTS (EID Sample Collected date/Échantillon EID Date de collecte)  (dd-MMM-yyyy)",
-	0 as "HTS Results/ Résultats HTS ",
-	0 as "Location of testing/ emplacement des tests",
-	0 as "Date Enrolled to HIV care / Date d'inscription aux soins du VIH (dd-MMM-yyyy)",
-	0 as "Linked to ART / Lié à ART",
-	"" as "Client ART Unique ID/Patient Unique ID/ART TARV",
-	0 as "ART Start Date/ Date de début de l'ART (dd-MMM-yyyy)",
-	0 as "Name of Facility where Treatment is provided/ Nom de l'établissement où le traitement est dispensé",
-	0 as "Reasons for Non-Linkage/ Raisons du non-couplage",
-	"" as "Time to Enrolment (days)",
-	"" as "Time to ART Initiation   (days)",
-	0 as "Date offered ICT / Date offerte ICT",
-	0 as "Date accepted ICT / Date d'acceptation ",
-	0 as "No of contacts elicited/ Nombre de contacts obtenus",
-	0 as "No of contacts notified / Nombre de contacts notifiés",
-	0 as "No of contacts tested for HIV/ Nombre de contacts testés pour le VIH",
-	0 as "No of contacts tested HIV positive/ Nombre de contacts testés séropositifs",
-	0 as "APS";
\ No newline at end of file
+	getPatientGender(p.patient_id) as "Sex",
+	getPatientARTNumber(p.patient_id) as "Index Case Code (ART Code/ Testing Code*)",
+	getIndexType(p.patient_id) as "Type of Index Case/ type de cas index",
+	getFirstIndexRelationship(p.patient_id) as "Index Classification / Classification de l'indice",
+	getTestingEntryPoint(p.patient_id) as "HTS Entry/ entrée  (Testing Modality/ modalités de test)",
+	getPatientMostRecentProgramAttributeValueFromName(p.patient_id, 'PROGRAM_MANAGEMENT_2_NOTIFICATION_DATE') as "Date of contact notification / Date de notification de contact",
+	getPatientMostRecentProgramAttributeCodedValueFromName(p.patient_id, 'PROGRAM_MANAGEMENT_1_NOTIFICATION_METHOD', 'en') as "Method of contact notification/ Méthode de notification de contact",
+	getPatientMostRecentProgramAttributeCodedValueFromName(p.patient_id, 'PROGRAM_MANAGEMENT_3_NOTIFICATION_OUTCOME', 'en') as "Outcomes of notifictaion / Résultats de la notification",
+	"N/A" as "HTS Client ID (EID or HEI Number)",
+	getHIVTestDate(p.patient_id, "#startDate#", "#endDate#") as "Actual date of visit for HTS/ Date réelle de visite pour HTS (EID Sample Collected date/Échantillon EID Date de collecte)  (dd-MMM-yyyy)",
+	getHIVResult(p.patient_id, "#startDate#", "#endDate#") as "HTS Results/ Résultats HTS ",
+	getObsLocation(p.patient_id, "e7a002be-8afc-48b1-a81b-634e37f2961c") as "Location of testing/ emplacement des tests",
```

Comment:
@lauravignoli is this the location of the index or contact?

Reply:
I think because of where it is in the sheet, it should be the contact testing location. It seems that all that info is grouped together. But we can check with Emeka later today.